src/Testcontainers.Xunit/MessageSinkLogger.cs

```diff
@@ -0,0 +1,24 @@
+namespace DotNet.Testcontainers.Xunit;
+
+internal sealed class MessageSinkLogger(IMessageSink messageSink) : ILogger, IDisposable
```

Comment:
Same here. We can simply use overloaded ctors and use the same underlying implementation to write the message (for `IMessageSink` and `ITestOutputHelper`). I can share the pattern that I usually use tomorrow. Then we can discuss this topic further.

Reply:
I finally went with a base `Logger` class in 610f7ba0013c7014d6cd02d1aa733c3d2c61ebec (Share code between logger implementations)